app/grants/migrations/0003_auto_20181010_2148.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.2 on 2018-10-10 21:48
```

Comment:
These migrations need to be deleted and recreated as a single migration at some point